nengo_loihi/snips/nengo_io.c.template

```diff
@@ -17,41 +25,100 @@ void nengo_io(runState *s) {
     int inChannel = getChannelID("nengo_io_h2c");
     int outChannel = getChannelID("nengo_io_c2h");
     int32_t count[1];
-    int32_t spike[2];
+    int32_t spike[3];
     int32_t error[2];
     int32_t output[N_OUTPUTS];
 
+    int32_t *value = (int32_t*)(s->userData+IDX_VALUES);
+    int32_t *pos_accum = (int32_t*)(s->userData+IDX_POS_ACCUMULATORS);
+    int32_t *neg_accum = (int32_t*)(s->userData+IDX_NEG_ACCUMULATORS);
+
     if (inChannel == -1 || outChannel == -1) {
         printf("Got an invalid channel ID\n");
         return;
     }
 
-    if (s->time % 100 == 0) {
+    if (s->time == 1) {
+        printf("initializing\n");
+        for (int i=0; i<N_INPUTS; i++) {
+            readChannel(inChannel, spike, 3);
+            s->userData[IDX_SPIKE_TARGETS+(i*3)+0] = spike[0];  // core id
+            s->userData[IDX_SPIKE_TARGETS+(i*3)+1] = spike[1];  // pos axon
+            s->userData[IDX_SPIKE_TARGETS+(i*3)+2] = spike[2];  // neg axon
+            printf("  spike target %d: (%d %d %d)\n",
+                   i, spike[0], spike[1], spike[2]);
+        }
+    }
+
+    if (s->time % 1000 == 0) {
         printf("time %d\n", s->time);
     }
 
-    readChannel(inChannel, count, 1);
+    //readChannel(inChannel, count, 1);
```

Comment:
Are these commented out lines still relevant or can we delete them?

Reply:
Definitely deleteable.  I'll do that.